### ISSUE TITLE
Run conditions prohibit running with no connection

### ISSUE THREAD

**Issue by imsodin** [syncthing/syncthing-android#1238]
When I am no on wifi and have mobile data disabled, the app will never run regardless of the set run conditions. My use case to run syncthing even though there will be no connection is scanning: It can take a long time if there is significant changed data, so I'd like that to happen even in between wifi connectivity.

There's two options that would work for me: Invert the run conditions, i.e. checkboxes meaning do not run on... Or add another option like "Always run" which when activated does just that and all other options are grayed out.

**Comment by Catfriend1** [comment]
Hi @imsodin
Can you work around the problem by "ticking run in flight more"? This was intended to cover the no connection case.

**Comment by imsodin** [comment]
@Catfriend1 Indeed, that works. In my opinion is definitely not intuitive (I looked at it but didn't even try it, as I was never in flight mode).

**Comment by Catfriend1** [comment]
Ok I'd suggest renaming that option by changing the language strings to make more clear that it covers flight mode and no network connected case.

**Comment by codethief** [comment]
I second this request. I'm operating a wifi hotspot which is *not* connected to the internet for good reason and would still like to use Syncthing to synchronize machines locally within this network. If it wasn't for @imsodin, I would still be wondering what's going on and would have reported the issue myself.
